decidim-core/lib/tasks/upgrade/clean.rake

```diff
@@ -140,6 +141,13 @@ namespace :decidim do
         logger.info("===== Updated #{blocked_users} blocked users")
       end
 
+      desc "Removes all the invalid records from private downloads"
+      task invalid_private_exports: :environment do
+        invalid_private_exports = Decidim::PrivateExport.where("export_type LIKE 'survey_user_responses_%'")
```

Comment:
The only edge case that I can come-up with is that the meta-task `decidim:upgrade:clean:invalid_records` may be called in a next release (like in v0.32 or v0.33) and it will remove valid records from PrivateExports. Maybe we can fine tune this query to check for the invalid name (I understand that this is the culprit)?

Reply:
As mentioned in the PR description, the names were generated like : `survey_user_responses_88ca03fab3b71024a7d97639bc10d62b771d69ae2c89d748055762e6b41d3e4d`  which caused an invalid I18n ... I could try to see if the key exists, and if is not to remove the key ... but this is not gonna fix the error if the user visits the page with a missing locale...

Reply:
@andreslucena , i have changed the query to : 
```
Decidim::PrivateExport.where("export_type ~ '^survey_user_responses_[0-9a-f]{64}$'")
```

Which aims to match only strings that have that sha format.

Reply:
Awesome!! I'll try that now